src/Tactic/Cong.agda

```diff
@@ -132,6 +146,21 @@ private
     `y ← quoteTC y
     pure $ def (quote cong) $ `a ⟅∷⟆ `A ⟅∷⟆ level ⟅∷⟆ type ⟅∷⟆ vLam "ϕ" f ⟨∷⟩ `x ⟅∷⟆ `y ⟅∷⟆ eq ⟨∷⟩ []
 
+  module _ where
+    private
+      applicative : ∀ {ℓ} → RawApplicative {ℓ} λ _ → List Meta
+      applicative = mkRawApplicative (λ _ → List Meta) (λ _ → []) _++_
+
+      open Traversal applicative
+
+      actions : Actions
+      actions = record defaultActions { onMeta = λ _ x → x ∷ [] }
+
+    blockOnMetas : Term → TC ⊤
+    blockOnMetas t with traverseTerm actions (0 , []) t
+    ... | []         = pure tt
+    ... | xs@(_ ∷ _) = blockTC (blockerAll (List.map blockerMeta xs))
```

Comment:
This feels like this (or part of it) could be extracted to a utility method in `Reflection.AST.Traversal` itself?

Reply:
I like the idea! Other tactics might want to block on metas, too. But please help me understand: The `Traversal` module takes an applicative as an argument. Wouldn't a user then have to provide a "dummy" applicative in order to use `blockOnMetas`? (Which would ignore the user-provided applicative.)

Reply:
What I'm trying to say: Should there be a new module, `Reflection.AST.Utilities`, that would be home to `blockOnMetas`?

Reply:
Ah sorry, I missed that `Traversal` is parameterised by an applicative. Yes a new module `Reflection.TCM.Utilities` would be good. (Note I've had a change of heart from `AST` to `TCM`!)

Reply:
Alright. Great. I moved it.